tests/tailsort/loadandsave/tests.csv

```diff
@@ -0,0 +1,11 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+before,0,0,q,.ts.hdbdir:`:/home/edalzell/cloudTorQ/TorQ/tests/tailsort/loadandsave/hdb,1,,"set test HDB path"
+before,0,0,q,.ts.taildir:`:/home/edalzell/cloudTorQ/TorQ/tests/tailsort/loadandsave,1,,"set test tailDB path"
```

Comment:
shouldn't have absolute file paths in tests, use env vars

Reply:
took these out and defined kdbtail and kdbhdb in run.sh instead